src/app.md

```diff
@@ -1,2 +1,2 @@
 /*TODO: Set app headers, including the CORS */
-# THE AP
\ No newline at end of file
+# THE AP IS HERE!!
```

Comment:
Nice header!

Reply:
There is a mistake in the word (AP). it should be (APP), please update it